api_open_fga.go

```diff
@@ -918,6 +921,16 @@ func (a *OpenFgaApiService) CheckExecute(r ApiCheckRequest) (CheckResponse, *_ne
 			return localVarReturnValue, nil, err
 		}
 
+		var otel = &telemetry.Telemetry{}
+		var attrs = otel.Metrics().AttributesFromRequest(req)
+		attrs[telemetry.FGAClientRequestMethod] = "Check"
+		attrs[telemetry.FGAClientRequestStoreID] = r.storeId
+		attrs[telemetry.HTTPRequestResendCount] = strconv.Itoa(i + 1)
```

Comment:
Can we make this start from 0, and not send it if the request has not been retried?

https://opentelemetry.io/docs/specs/semconv/http/http-spans/#http-client-retries-examples

Also here
<img width="819" alt="Screenshot 2024-07-30 at 10 50 05 AM" src="https://github.com/user-attachments/assets/afe38de6-2a62-4cf1-8fbe-8770d0d7fed1">

Reply:
@rhamzeh Sure thing, easy change! I've updated the PR